package.json

```diff
@@ -85,57 +82,61 @@
     "lodash.escaperegexp": "^4.1.2",
     "lodash.kebabcase": "^4.1.1",
     "lodash.startcase": "^4.4.0",
-    "postcss": "^6.0.21",
     "prop-types": "^15.5.8",
     "react-intl": "^2.4.0",
-    "terra-base": "^5.0.0",
+    "react-router-dom": "^5.0.0",
     "terra-action-header": "^2.2.0",
-    "terra-application-layout": "^4.0.0",
+    "terra-aggregate-translations": "^1.0.0",
+    "terra-application-layout": "^5.1.0",
+    "terra-base": "^5.0.0",
     "terra-button": "^3.2.0",
     "terra-content-container": "^3.0.0",
     "terra-doc-template": "^2.2.0",
     "terra-icon": "^3.0.0",
     "terra-image": "^3.0.0",
     "terra-markdown": "^2.0.0",
-    "terra-navigation-layout": "^4.0.0",
     "terra-navigation-side-menu": "^2.0.0",
     "terra-search-field": "^3.2.0",
     "terra-slide-panel": "^3.0.0",
     "terra-theme-provider": "^3.0.0",
-    "terra-toolkit": "^4.5.0",
     "to-ast": "^1.0.0",
     "webpack-merge": "^4.1.2",
     "xfc": "^1.2.1"
```

Comment:
Can we remove XFC in this MVB and remove the XFC Provider initialization here: https://github.com/cerner/terra-dev-site/blob/master/src/Index.jsx#L11

People will be able to define their own if they need it and import the XFC initialization via side-affects

Reply:
yeah we could do that here.